public/editor-client/src/aiText/index.ts

```diff
@@ -0,0 +1,104 @@
+import { request } from "../api";
+import { getConfig } from "../config";
+import { Action, Conversation, RequestOptions, Role } from "../types/AiText";
+
+export const doAiRequest = async (
+  res: (value: string) => void,
+  rej: (error: string) => void,
+  data: { prompt: string; action?: Action }
+): Promise<void> => {
+  const config = getConfig();
+
+  if (!config) {
+    throw new Error("Invalid __BRZ_PLUGIN_ENV__");
+  }
+
+  try {
+    const { prompt, action } = data;
+    const conversation: Conversation[] = [];
+
+    switch (action) {
+      case Action.Extend:
+        conversation.push({
+          role: Role.User,
+          content: "Expand the following text by 2 times: "
```

Comment:
need to be translatable

Reply:
No, because `content` not part of ui. It's request for AI and should be on english